admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/server/dao/DynamicEntityDaoImpl.java

```diff
@@ -153,6 +153,9 @@
 
     protected ApplicationContext applicationContext;
 
+
+    protected FieldManager fieldManager = new FieldManager(entityConfiguration, null);
```

Comment:
Why not start this as `null` and make it behave the same way as the `standardEntityManager`? It looks like you are intending them to have the same semantics and they are essentially tied together.

Reply:
Hmm, don't rembmer, I think main reason behind that if in some new place a `DynamicEntityDaoImpl` will be created it is not really necessary to call the `setStandardEntityManager` method, and potentially we might have NPE... I can probably move this instatiation to `getFieldManager()` method but not sure what I don't like more setters or getters that do some extra work.
Maybe DynamicEntityDao should have a constructor for that field, but I'm not really sure about all the places where and how it can/will be used, so tried to do non-destructive change
But you are right, the main reason is to tight `standardEntityManager` and `fieldManager` , so let me know if I should default it to null and maybe move "check for null and instance creation as a  protection" to getter. Just in case

Reply:
Yeah I think default to null and move the null-checking protection in the getter makes the most sense.